src/components/__tests__/Button.test.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import Buttons from '../Buttons'; // Adjust the path
+
+describe('Buttons component', () => {
+  it('should render with the provided number and handle event', () => {
+    const handleEventMock = jest.fn();
+
+    const tree = renderer
+      .create(<Buttons number="1" handleEvent={handleEventMock} />)
+      .toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+
+  // Add more test cases as needed
+});
```

Comment:
- [ ] Great work using Jest snapshots and the `react-test-renderer` package to test components. In addition to these tests, kindly use **_React Testing Library_** to simulate user interaction for your application and components.👍. You can learn how to use **_React Testing Library_** by reading these articles:
- https://testing-library.com/docs/react-testing-library/intro/
- https://www.robinwieruch.de/react-testing-library/

### Project Requirement

![image](https://github.com/RicardoEgas/Math-Magicians/assets/77106962/976765be-3945-4f78-b3f3-bcf2cb8fff32)